models/DataObject/ClassDefinition/Data/Multihref.php

```diff
@@ -721,7 +721,12 @@ public function preGetData($object, $params = [])
 
                 $setter = 'set' . ucfirst($this->getName());
                 if (method_exists($object, $setter)) {
```

Comment:
same here ... why check for $setter exists if we don't call it anymore?

Reply:
same as above